gulpfile.js

```diff
@@ -370,4 +371,52 @@ gulp.task('idea', 'Setup project files for IDEA / PhpStorm', function () {
 
 function isRelative(sitePath) {
     return path.normalize(sitePath) != path.resolve(sitePath);
+}
+
+/**
+ * Compress the build into a zip file.
```

Comment:
There should be an explanation here why we're not using the `gulp-zip` package – a link to some issue or something.